### DISCUSSION TITLE
Question about scenario Grace Gravity field Recovery

### DISCUSSION THREAD

**Discussion by RoyanLiSu** [groops-devs/groops#149] [Q&A] [answered]
Dear Developers,
in this scenario we use "simulate satellite tracking" to detect outliers from original SST observations. why the difference between the simulated data and the observations are so large, especially for the range data? 
the simulated satellite tracking data was built from the approximate orbits (GNV1B), is it reasonable to have such a difference with KBR1b data?
I would be very grateful if you could explain this to me.
kind regards

![qu1](https://user-images.githubusercontent.com/110038504/218308341-b0bbe101-30d0-4349-a58c-4e041181840d.png)
![qu2](https://user-images.githubusercontent.com/110038504/218308352-47658f1a-4922-4e7d-a4b8-8c4c3c45c673.png)

**Comment by tmayerguerr** [comment, marked as answer]
Dear  RoyanLiSu ,

The KBR instrument is not able to measure the absolute range but only range changes. Therefore the range contains an unknown constant (until the next phase break) bias. Additionally you have to consider (to add) the light time correction and the antenna center correction to the observations.

regards
Torsten

**Comment by RoyanLiSu** [comment]
thank you so much for your response I understand it. so I can't compare these columns for the range data. 
but what about the range rate data? it seems the difference between them is about 1e-6. 
kind regards,
Royan

**Reply by tmayerguerr** [reply]
Yes, we usually use the range-rate data for gravity field recovery. The accuracy is slightly better than 1e-6 m/s.
Torsten

**Reply by RoyanLiSu** [reply]
The matter became clear to me thank you for your explanation.
with kind regards,
Royan
